coalib/bearlib/languages/Language.py

```diff
@@ -429,14 +471,105 @@ def limit_versions(language, limit, operator):
         versions = [version for version in language.versions
                     if operator(int(version), limit)]
     else:
-
         versions = [version for version in language.versions
                     if operator(version, limit)]
     if not versions:
         raise ValueError('No versions left')
     return type(language)(*versions)
 
 
+def type_equals(language, comparable):
```

Comment:
Better name it `check_type_equals` or `assert_type_equals` to make its functionality a bit clearer